.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,72 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
```

Comment:
I don't think we want to use this label since everything in the issue tracker that a user reports can be described as a bug

Reply:
Would it be evil to use the Incomplete label until a maintainer/somebody comes in and properly confirms/labels it?

Reply:
We'll except for feature requests... 
But yeah bug is used way more often so I get why that might just add noise.